### ISSUE TITLE
There are so many duplicates in my prediction

### ISSUE THREAD

**Issue by jyrgogo** [kenchan0226/keyphrase-generation-rl#4]
I trained a catSeq model and used it to generate keyphrases, but the results contain many duplicates, I don't know why, can you give me some suggestion? The results are shown bellow.
![result](https://user-images.githubusercontent.com/44525934/66895056-7988ad00-f024-11e9-9fa7-7523f1f919df.png)

**Comment by kenchan0226** [comment]
Yes, I found that sometimes the original catSeq model contains duplicates. Can you check if you can get similar F1 scores of catSeq that is reported in our paper? If you can get similar F1, then you can use reinforcement learning to fine-tune the pre-trained model, it will significantly reduce the duplications since the reward function will treat those duplications as incorrect predictions.

**Comment by jyrgogo** [comment]
I'll try it, thank you very much!
